### PR TITLE
Remove codecov/project step

### PR DESCRIPTION
**Pull request by victorschumacherzup**
### Description

The codecov/project check is no longer making sense
We need to constantly update the branch because master is always changing

image

### Changelog
Disable codecov/project check

### CODE REVIEW

**Review by lucasdittrichzup — APPROVED** [review]

File: pkg/formula/runner/local/pre_run_test.go [hunk 1]
```diff
@@ -180,33 +180,7 @@ func TestPreRun(t *testing.T) {
 				err:     errors.New("error to create dir"),
 			},
 		},
-		{
-			name: "copy work dir error",
```
Comment (henriquemoraeszup) [inline pkg/formula/runner/local/pre_run_test.go:184]: what is this change for?

**Review by henriquemoraeszup — APPROVED** [review]
YES

File: pkg/formula/runner/local/pre_run_test.go (hunk 1 above)
Comment (victorschumacherzup) [reply pkg/formula/runner/local/pre_run_test.go:184]: just removing it to test the codecov, i will put it back after finishing

Comment (victorschumacherzup) [reply pkg/formula/runner/local/pre_run_test.go:184]: i just request reviewers too early, sry

Comment (henriquemoraeszup) [reply pkg/formula/runner/local/pre_run_test.go]: no worries
=]

**Review by henriquemoraeszup — APPROVED** [review]

**Review by lucasdittrichzup — APPROVED** [review]

**Review by brunasilvazup — APPROVED** [review]

**Comment by henriquemoraeszup** [comment]
I like how the explanation is phrased under **Description** 😃

**Pull request merged**